exampleCourse/courseInstances/Sp15/assessments/hw06-demoPythonAutograder/infoAssessment.json

```diff
@@ -35,12 +35,14 @@
             "questions": [
                 {"id": "demo/autograder/codeEditor",            "points": 1, "maxPoints": 5},
                 {"id": "demo/autograder/codeUpload",            "points": 1, "maxPoints": 5},
-                {"id": "demo/autograder/python/square",                "points": 1, "maxPoints": 5},
-                {"id": "demo/autograder/python/numpy",                 "points": 1, "maxPoints": 5},
-                {"id": "demo/autograder/python/pandas",                "points": 1, "maxPoints": 5},
-                {"id": "demo/autograder/python/plots",                 "points": 1, "maxPoints": 5},
-                {"id": "demo/autograder/python/random",                "points": 1, "maxPoints": 5},
-                {"id": "demo/autograder/ansiOutput",                      "points": 2, "maxPoints": 4}
+                {"id": "demo/autograder/python/square",         "points": 1, "maxPoints": 5},
+                {"id": "demo/autograder/python/numpy",          "points": 1, "maxPoints": 5},
+                {"id": "demo/autograder/python/pandas",         "points": 1, "maxPoints": 5},
+                {"id": "demo/autograder/python/plots",          "points": 1, "maxPoints": 5},
+                {"id": "demo/autograder/python/random",         "points": 1, "maxPoints": 5},
+                {"id": "demo/autograder/ansiOutput",            "points": 2, "maxPoints": 4},
+                {"id": "demo/autograder/python/orderBlocks_randomParams",      "points": 1, "maxPoints": 2},
```

Comment:
I'm not a big fan of mixing camelCase and snake_case. Maybe we should go with `orderBlocksRandomParams` for consistency?

Reply:
fixed